src/mem/cache/prefetch/Prefetcher.py

```diff
@@ -228,6 +228,38 @@ class TaggedPrefetcher(QueuedPrefetcher):
     degree = Param.Int(2, "Number of prefetches to generate")
 
 
+class IpcpPrefetcher(QueuedPrefetcher):
+    # Paper: https://doi.org/10.1109/ISCA45697.2020.00021
```

Comment:
Can you add an `add_citation` call for this. See https://github.com/gem5/gem5/blob/stable/src/mem/MemCtrl.py#L106 for an example.